packages/autotask-client/src/api.ts

```diff
@@ -85,9 +90,17 @@ export class AutotaskClient extends BaseApiClient {
     return this.updateCode(autotaskId, encodedZippedCode);
   }
 
-  public async listAutotaskRuns(autotaskId: string, next?: string): Promise<AutotaskRunListResponse> {
+  public async listAutotaskRuns(
+    autotaskId: string,
+    next?: string,
```

Comment:
Wondering what "next" is? Should it be typed AutotaskRunStatus here?

maybe we could combine `next === 'success' || next === 'error' || next === 'pending' || next === 'throttle')` and `next as AutotaskRunStatus;` as a function using typescript "is" ?

Reply:
`next` is dynamo paging cursor in this context.